client/a8c-for-agencies/components/user-feedback-modal-form/index.tsx

```diff
@@ -0,0 +1,157 @@
+
+import './style.scss';
+
+type Props = {
+	show: boolean;
+	onClose?: () => void;
+};
+
+const DEFAULT_FEEDBACK_VALUE = '';
+const DEFAULT_RATING_VALUE = 0;
+
+export default function UserFeedbackModalForm( { show, onClose }: Props ) {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const [ feedback, setFeedback ] = useState( DEFAULT_FEEDBACK_VALUE );
+	const [ rating, setRating ] = useState( DEFAULT_RATING_VALUE );
+
+	const { isSubmittingFeedback, submitFeedback, isSubmissionSuccessful } =
+		useSubmitProductFeedback();
+
+	const onModalClose = useCallback( () => {
+		setFeedback( DEFAULT_FEEDBACK_VALUE );
+		setRating( DEFAULT_RATING_VALUE );
+		onClose?.();
+
+		dispatch( recordTracksEvent( 'calypso_a4a_user_feedback_form_close' ) );
+	}, [ dispatch, onClose ] );
+
+	useEffect( () => {
+		if ( isSubmissionSuccessful ) {
+			dispatch(
+				successNotice( translate( 'Thank you for your feedback!' ), {
+					id: 'submit-product-feedback-success',
+					duration: 5000,
+				} )
+			);
+			onModalClose();
+		}
+	}, [ dispatch, isSubmissionSuccessful, onModalClose, translate ] );
+
+	const onFeedbackChange = useCallback( ( event: ChangeEvent< HTMLInputElement > ) => {
+		setFeedback( event.currentTarget.value );
+	}, [] );
+
+	const onRatingChange = useCallback(
+		( rating: number ) => {
+			dispatch( recordTracksEvent( 'calypso_a4a_user_feedback_form_rating_click' ) );
+			setRating( rating );
+		},
+		[ dispatch ]
+	);
+
+	const hasCompletedForm = !! feedback && !! rating;
+
+	const onSubmit = useCallback( () => {
+		if ( ! hasCompletedForm ) {
+			return;
+		}
+
+		dispatch(
+			recordTracksEvent( 'calypso_a4a_user_feedback_form_submit', {
+				rating,
+				feedback,
+			} )
+		);
+
+		const sourceUrl = `${ window.location.origin }${ window.location.pathname }`;
+		submitFeedback( { feedback, rating, source_url: sourceUrl } );
+	}, [ dispatch, feedback, hasCompletedForm, rating, submitFeedback ] );
+
+	useEffect( () => {
+		if ( show ) {
+			dispatch( recordTracksEvent( 'calypso_a4a_user_feedback_form_open' ) );
+		}
+	}, [ dispatch, show ] );
+
+	if ( ! show ) {
+		return null;
+	}
+
+	return (
+		<Modal
+			className="user-feedback-modal-form"
+			onRequestClose={ onModalClose }
+			__experimentalHideHeader
+		>
+			<div className="user-feedback-modal-form__main">
+				<Button
+					className="user-feedback-modal-form__close-button"
+					plain
+					onClick={ onModalClose }
+					aria-label={ translate( 'Close' ) }
+				>
+					<Icon size={ 24 } icon={ close } />
+				</Button>
+
+				<h1 className="user-feedback-modal-form__title">
+					{ translate( 'Help us make Automattic for Agencies even better' ) }
+				</h1>
+
+				<p className="user-feedback-modal-form__instruction">
+					{ translate(
+						'Your feedback is extremely valuable to us. All feedback is set to our product and program teams, and helps to inform how we evolve our offering.'
```

Comment:
Is "set" supposed to be "sent"? --> All feedback is sent to our product and program teams...

Reply:
Thanks for flagging this! Fixed in https://github.com/Automattic/wp-calypso/pull/90632

Reply:
Sorry for the late response.

Thanks @emilyaudela and @vitozev for reporting and fixing the issue 🙏